symbols/overlay29.yml

```diff
@@ -495,10 +599,24 @@ overlay29:
       address:
         NA: 0x22F97F0
         EU: 0x22FA1FC
+        JP: 0x22FADA8
       description: |-
         Checks if an entity has the ability Truant, and if so tries to apply the pause status to it.
         
         r0: pointer to entity
+    - name: MonsterCanSpawn
```

Comment:
Oh right, I didn't remember that, I was thinking about the mystery part/slab spawns but mew wasn't among those. This would actually make sense, so what about renaming it something like `IsUnusedMewSpawn`? Considering that it would return true for all other monster IDs

Reply:
What do you mean by "unused" in `IsUnusedMewSpawn`? Do we know that it's actually unused? Maybe we can call it something like `MewSpawnCheck`.

Reply:
Yeah that's better, I just assumed that the check is never used but we don't actually know for sure